src/hooks/core/chat/useFetchChannelHook.ts

```diff
@@ -16,19 +17,7 @@ type ChannelCategory = 'SIGNED' | 'ANONYMOUS';
 const useFetchChannelHook = () => {
   const {localDb, refresh} = useLocalDatabaseHook();
   const {signedProfileId, anonProfileId} = useUserAuthHook();
-
-  const helperFollowingSystemMessage = (message) => {
-    if (message?.system_user === signedProfileId) {
-      message.text = message?.own_text;
-    } else {
-      message.text = message?.other_text;
-    }
-    /**
-     * TODO: Handle follow anonymous user system message later, if any
-     * */
-
-    return message;
-  };
+  const {getFirstMessage} = useSystemMessage();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The usage of `getFirstMessage` function from `useSystemMessage` hook is a good change. It improves the readability of the code by abstracting away the details of how the first message is retrieved.
```diff
+ const {getFirstMessage} = useSystemMessage();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->